requirements.txt

```diff
@@ -9,3 +9,5 @@ papermill
 gitpython
 black==19.10b0
 black-nb
+seaborn
```

Comment:
please pip install these (and lightgbm itself) in the notebook - see some other examples which have pip install cells

Reply:
Thanks for checking this. Let me make the corresponding change and update the PR.